### DISCUSSION TITLE
How to specify organization when login or registration?

### DISCUSSION THREAD

**Discussion by tunhvn** [zitadel/zitadel#5804] [Q&A] [answered]
Currently I am using Keycloak, I can specify realm with customized branding (logo, external IdP...) for my users using URL.
Each realm has it own URL.
How can I do this with organization in Zitadel? Is there any option for customized URL (for customized branding) as Keycloak?

**Comment by mffap** [comment, marked as answer]
Hey @tunhvn 

Two ways in ZITADEL. 
Firstly, you can use the authentication request and pass the organization in one of the scopes. Check out our [playground](https://zitadel.com/docs/apis/openidoauth/authrequest) and you can try it out directly. We have also a more [detailed blog ](https://zitadel.com/blog/use-oidc-auth-request)on how to customize the experience with the oidc auth request.
Secondly you can also use [Domain Discovery](https://zitadel.com/docs/guides/solution-scenarios/domain-discovery) to do the routing to the correct domain (incl. applying all the settings for said domain) automatically.
